Google-Summer-of-Code-2023.md

```diff
@@ -0,0 +1,261 @@
+
+-   [Google Summer of Code contributor guide](https://google.github.io/gsocguides/student/)
+-   [Google's list of resources](https://developers.google.com/open-source/gsoc/resources/)
+-   [GSoC FAQ](https://developers.google.com/open-source/gsoc/faq)
+
+
+## Getting started
+
+Welcome! If you're interested in applying to work with Oppia for GSoC, please follow these steps:
+
+1. Sign up to the [oppia-gsoc-announce@](https://groups.google.com/forum/#!forum/oppia-gsoc-announce) mailing list in order to receive important notifications about Oppia's participation in GSoC. If you like, you can turn on the notification for github discussions to  participate in general discussion related to Oppia's involvement in GSoC. Make sure to set your preferences correctly so that you actually get the emails!
+
+2. Get a better understanding of what Oppia is about:
+    - Read the [user documentation](http://oppia.github.io/#/) to become familiar with important concepts like explorations and interactions.
+    - Play some lessons on [Oppia.org](https://www.oppia.org/learn/math), which hosts a live instance of Oppia.
+
+3. To get started with development, read and follow the instructions in the contributors' guide carefully ([Oppia Web](https://github.com/oppia/oppia/wiki/Contributing-code-to-Oppia#setting-things-up)).
+
+4. Do one or more starter projects to become familiar with the contribution process. This will help us get an idea of what it's like to work with you. It will also help you get a better understanding of the codebase and our development process, which may help with writing a good project proposal. Once you've merged at least 2 pull requests, you will get an invitation to become a collaborator to the Oppia or Oppia-Android repository and be officially onboarded!
+
+   - **Pro-tip!** Quality is more important than quantity; we want to see examples of your best work. So, please make sure to follow the [tips for success](https://github.com/oppia/oppia/wiki/Contributing-code-to-Oppia#tips-for-success), manually test your code before submitting (to ensure it does what you want it to and doesn't break anything else), ensure that your code conforms to the [style rules](https://github.com/oppia/oppia/wiki/Coding-style-guide), and pay attention to small details. These are good skills to learn when developing software in general, and they will also help you build credibility as a responsible developer who can be trusted to be a good steward of the Oppia codebase.
+
+5. Select one or more [GSoC project ideas](#oppias-project-ideas-list) that you're most interested in, and write your project proposal! We strongly encourage you to discuss your project ideas and share your proposal with the community, so that you can get feedback and ensure that what you're writing makes sense to others. Please follow the instructions in the [GSoC proposal template](#gsoc-proposal-template) section to do this.
```

Comment:
Should we specify that being onboarded is one of our requirements for GSoC?

Reply:
Added that to point 4